src/css/tabs/motors.css

```diff
@@ -471,19 +471,8 @@
     .tab-motors .motor_testing .telemetry li {
         font-size: 6px;
     }
-    .tab-motors #dialogMotorOutputReorder {
-        position: fixed;
-        width: calc(100% - 2em);
-        bottom: 0;
-        top: 56px;
-        border-radius: unset;
-        border: none;
-        overflow: auto;
-        margin: 0;
-        height: auto;
-    }
-
-    .tab-motors #escDshotDirectionDialog- {
+    .tab-motors #dialogMotorOutputReorder,
+    .tab-motors #escDshotDirectionDialog {
```

Comment:
Seeing that this can grow and this seems to be a common style, maybe is better to have something like:

`.tab-motors .dialog {` (or `.tab-motors .motorDialog {` if `.dialog` is being used with other styles) and add the `dialog` class to the html.

In this way is more maintainable (we don't need to touch the css, only the html) and we limit the adhoc styles.